flytekit/models/task.py

```diff
@@ -872,6 +874,14 @@ def image(self):
         """
         return self._image
 
+    @property
+    def architecture(self):
```

Comment:
```suggestion
    def architecture(self) -> int:
```
or `typing.Optional[int]`

Reply:
Done